frontend-cert/react-projects/react-use-state/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
I don't think we need any of the comments you added here because the code is pretty self explanatory